.gitignore

```diff
@@ -8,3 +8,8 @@
 _build/
 build/
 dist/
+
+# PyCharm
```

Comment:
Can you add these without the comments in sorted order?

Reply:
can do